server/src/main/java/org/apache/uniffle/server/netty/ShuffleServerNettyHandler.java

```diff
@@ -454,7 +454,7 @@ public void handleGetLocalShuffleData(TransportClient client, GetLocalShuffleDat
             requestInfo);
         response =
             new GetLocalShuffleDataResponse(
-                req.getRequestId(), status, msg, sdr.getManagedBuffer());
+                req.getRequestId(), status, msg, new NettyManagedBuffer(sdr.getManagedBuffer().byteBuf()));
```

Comment:
We know the managedBuffer is `FileSegmentManagedBuffer` . To keep performance, we should ensure the `FileChannel` is used, which is invoked by `convertToNetty` method. Right?